testing.go

```diff
@@ -80,3 +80,48 @@ func newFlakyServer(uri string, code int, nbFlakyResponses int) *flakyServer {
 	server := httptest.NewServer(http.HandlerFunc(handler))
 	return &flakyServer{server, &nbRequests, &requests}
 }
+
+type simpleResponse struct {
+	status int
+	body   string
+}
+
+type SimpleTestServer struct {
+	*httptest.Server
+
+	responses     map[string][]simpleResponse
+	responseIndex map[string]int
+}
+
+func NewSimpleServer() *SimpleTestServer {
+	server := &SimpleTestServer{
+		responses:     make(map[string][]simpleResponse),
+		responseIndex: make(map[string]int),
+	}
+	server.Server = httptest.NewUnstartedServer(http.HandlerFunc(server.handler))
+	return server
+}
+
+func (s *SimpleTestServer) addResponse(path string, status int, body string) {
```

Comment:
I do feel though that this testing simple server should end up in the testing repository not gomaasapi publicly but later...